packages/dcos-integration-test/extra/test_remove_dockercfg_hook.py

```diff
@@ -0,0 +1,30 @@
+import uuid
+
+import pytest
+
+
+def test_remove_dockercfg_hook(dcos_api_session):
+    """Test that the remove .dockercfg hook is working properly.
+
+    If the hook is enabled, the test expects that the .dockercfg file
+    (downloaded as a uri via the Mesos fetcher) is removed from the task's sandbox.
+
+    """
+
+    # Skip the test if the hook is disabled
+    if not dcos_api_session.dockercfg_hook_enabled:
+        pytest.skip('Test requires dockercfg hook to be enabled')
+
+    # Create a one-off job checking that the fetched .dockercfg file is not in the sandbox
+    job = {
+        'id': 'integration-test--' + uuid.uuid4().hex,
+        'run': {
+            'cpus': .1,
+            'mem': 32,
+            'disk': 0,
+            'cmd': "test ! -f .dockercfg",
+            'artifacts': [{'uri': "file:///opt/mesosphere/active/dcos-integration-test/util/.dockercfg"}]}}
+    try:
```

Comment:
No need to try/except. If the task fails it will raise an exception and this test will fail without needing to do so explicitly